rrule/src/iter/iterinfo.rs

```diff
@@ -257,6 +259,7 @@ impl<'a> IterInfo<'a> {
         }
     }
 
+    #[inline]
```

Comment:
How useful are these inline attributes really? I think I read somewhere that the compiler is usually able to inline things itself, and explicit inlining is only needed in rare cases.